DataConnectors/JumpCloud Single Sign On/AzureFunctionJumpCloud/JCQueueTrigger1/run.ps1

```diff
@@ -0,0 +1,141 @@
+<#  
+    Title:          JumpCloud Data Connector
+    Language:       PowerShell
+    Version:        1.0.1
+    Author(s):      Microsoft - Chris Abberley
+    Last Modified:  2020-08-03
+    Comment:        First Release
+
+    DESCRIPTION
+    This Function App calls the JumpCloud Directory Insights API (https://jumpcloud-insights.api-docs.io/1.0/api-overview/directory-insights) to pull the JumpCloud logs. The response from the JumpCloud API is recieved in JSON format. This function will build the signature and authorization header 
+    needed to post the data to the Log Analytics workspace via the HTTP Data Connector API. The Function App will post the JumpCloud logs to the JumpCloud_CL table in the Log Analytics workspace.
+#>
+
+# Input bindings are passed in via param block.
+param([string] $QueueItem, $TriggerMetadata)
+
+# Write out the queue message and insertion time to the information log.
+Write-Output "JumpCloud: Queue trigger for work item: $QueueItem, Queue item insertion time: $($TriggerMetadata.InsertionTime)"
+
+import-module AzTable
+
+# Retrieve Environment Variables and prep other Variables for the JumpCloud API request
```

Comment:
Please update azure gov related changes. see sample PR for reference.[https://github.com/Azure/Azure-Sentinel/pull/1752](https://github.com/Azure/Azure-Sentinel/pull/1752)